src/Microsoft.SqlTools.ServiceLayer/ObjectManagement/ObjectTypes/ServerRole/ServerRoleInfo.cs

```diff
@@ -0,0 +1,18 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+namespace Microsoft.SqlTools.ServiceLayer.ObjectManagement
+{
+    /// <summary>
+    /// a class for storing various server role properties
+    /// </summary>
+    public class ServerRoleInfo : SqlObject
+    {
+        public string? Owner { get; set; }
```

Comment:
this is missing the owner type information, it could be server role or login

Reply:
Yes that's correct, something like this will just work
```
    var serverRole = new ServerRole(srv, "cliRole");
    serverRole.Owner = "login name or server role name";
    serverRole.Create();
```